pkg/cmd/run/cmd.go

```diff
@@ -0,0 +1,117 @@
+// Copyright 2019 Tetrate
```

Comment:
seems inconsistent to me because other top commands are in `pkg/cmd` directory, and  only this `run` cmd has dedicated directory `pkg/cmd/run`.

Reply:
mainly I was trying to keep the PR from getting more massive. there was a circular dependency this resolved. (you'll notice the extension commands are in different dirs, but also there are almost no tests for the base commands either). Ack this is inconsistent, but I think we can do this after the fact as it will add a lot of code to backfill the missing tests. trust me to follow-up?

Reply:
it is to unwind an import cycle needed to share code since getenvoy run and getenvoy extension run share logic, since we mentioned earlier, I'll leave it to that comment.